tests/ci/run_cryptofuzz.sh

```diff
@@ -30,15 +44,21 @@ export CFLAGS="${CFLAGS} -I $OPENSSL_INCLUDE_PATH"
 export CXXFLAGS="${CXXFLAGS} -I $OPENSSL_INCLUDE_PATH"
 export LIBFUZZER_LINK="-fsanitize=fuzzer"
 
-# Build the overall cryptofuzz binary
+# Build the overall cryptofuzz and generate_corpus binary
 cd "$CRYPTOFUZZ_SRC"
-make "-j${NUM_CPU_THREADS}" cryptofuzz
+rm -rf cryptofuzz
+rm -rf generate_corpus
+make "-j${NUM_CPU_THREADS}" cryptofuzz generate_corpus
 
 # Common AWS-LC fuzzing setup, the cryptofuzz binary is in this folder so FUZZ_TEST_PATH=FUZZ_NAME
 FUZZ_NAME="cryptofuzz"
 FUZZ_TEST_PATH="${CRYPTOFUZZ_SRC}/${FUZZ_NAME}"
 SRC_CORPUS="$CRYPTOFUZZ_SEED_CORPUS"
 
+# For cryptofuzz development only, uncomment below code to generate new corpus.
+# rm -rf "$CRYPTOFUZZ_SEED_CORPUS" && mkdir "$CRYPTOFUZZ_SEED_CORPUS"
+# ./generate_corpus "$CRYPTOFUZZ_SEED_CORPUS"
+
```

Comment:
Is `CRYPTOFUZZ_SEED_CORPUS` defined in the environment prior to this script running?  I don't see it previously used in this script. (But I see it used in `build_cryptofuzz_modules.sh`.)

Reply:
Yes, it's created during Docker image build using `build_cryptofuzz_modules.sh` and then stored in `fuzz_env.sh`.
```sh
source "${FUZZ_ROOT}/fuzz_env.sh"
```

Result of `fuzz_env.sh`.
```sh
/home/dependencies# cat fuzz_env.sh
export CRYPTOFUZZ_SEED_CORPUS="/home/dependencies/cryptofuzz_seed_corpus"
export CRYPTOFUZZ_DICT="/home/dependencies/cryptofuzz-dict.txt"
export CFLAGS="-fsanitize=address,undefined,fuzzer-no-link -O2 -g"
export CXXFLAGS="-fsanitize=address,undefined,fuzzer-no-link -D_GLIBCXX_DEBUG -O2 -g"
export CRYPTOFUZZ_SRC="/home/dependencies/cryptofuzz"
```

> each script clearly documented what environment variables it expects to be set in its environment prior to starting.

`fuzz_env.sh` helps define all env variables before starting. Let me know if we want more documents.